### ISSUE TITLE
Downloads using civitai always download in the default directories

### ISSUE THREAD

**Issue by marouane-erouhi** [etherealxx/batchlinks-webui#15]
Downloads using civitai always download in the default directories regardless of any prior commands; eg:
```
@new #pokemon_loras /content/stable-diffusion-webui/models/Lora/Pokemon
#pokemon_loras
https://civitai.com/models/8243/misty-pokemon-lora-8-mb
```
This code should create the `Pokemon` directory and download into it.

**Comment by etherealxx** [comment]
ah, yes. The civitai model page link method (the link without '/api/' on it) automatically assign the directory to their default folder based on the tag on the CivitAi website. I should thought of that if someone use custom folder😅

thanks for the report, will be fixed soon.

**Comment by etherealxx** [comment]
fixed. Will update soon. If this issue is closed and the version is v3.0.3, then the issue is fixed.

![image](https://user-images.githubusercontent.com/64251396/226823116-dfc9d162-e24f-4b36-8841-a52929d2c18f.png)

**Comment by etherealxx** [comment]
The update is live (v3.1.0)! You can try it by yourself!
